# DP: Temporary work around:
# DP: On arm-linux-gnueabi run the libstdc++v3 testsuite with -Wno-abi

Index: b/src/libstdc++-v3/testsuite/lib/libstdc++.exp
===================================================================
--- a/src/libstdc++-v3/testsuite/lib/libstdc++.exp
+++ b/src/libstdc++-v3/testsuite/lib/libstdc++.exp
@@ -288,6 +288,10 @@
     }
     append cxxflags " "
     append cxxflags [getenv CXXFLAGS]
+    # ARM C++ emits an ABI warning for varargs.
+    if [istarget "arm*"] {
+        append cxxflags " -Wno-abi"
+    }
     v3track cxxflags 2
 
     # Always use MO files built by this test harness.
